ceph-ansible-pull-requests/config/definitions/ceph-ansible-pull-requests.yml

```diff
@@ -59,3 +59,11 @@
           !include-raw:
             - ../../../scripts/build_utils.sh
             - ../../build/build
+
+    publishers:
+      - github-notifier
+      - postbuildscript:
+          script-only-if-succeeded: True
+          script-only-if-failed: True
+          builders:
+            - shell: "curl -X POST https://mita.ceph.com/api/nodes/${NODE_NAME}/delete/"
```

Comment:
I think mita might need some work here to support a jenkins $NODE_NAME and not a mita UUID.